README.md

```diff
@@ -1,191 +1,107 @@
-# AI Code Reviewer
+# GPT TECH LEADER GIT REVIEWER
+
+[English Version](README_EN.md)
 
 AI 驱动的代码审查助手，自动为 GitHub Pull Requests 提供代码审查和建议。
 
 ## 功能特点
 
-- 自动代码审查：当 PR 创建或更新时自动触发审查
+> [!NOTE]  
+> 可以在本repo提MR, 体验功能, 样例MR(MR: https://github.com/IcyCC/gpt-techleader-git-reviewer/pull/5)
+
+- 自动代码审查：当 PR 创建或更新时自动触发, 对代码风格和业务逻辑进行审查
 - 智能评论回复：对开发者的评论进行智能回复
-- 代码质量分析：检查代码风格、潜在问题和改进建议
-- 完整的讨论支持：支持评论追踪和问题解决
+- 支持github
 
 ## 快速开始
 
-### 环境要求
-
-- Python 3.8+
-- Redis
-- GitHub 账号和 Personal Access Token
-
-### 安装步骤
-
 1. 克隆仓库：
 ```bash
-git clone https://github.com/your-username/ai-code-reviewer.git
-cd ai-code-reviewer
+git clone https://github.com/IcyCC/gpt-techleader-git-reviewer.git
+cd gpt-techleader-git-reviewer
 ```
 
-2. 创建并激活虚拟环境：
+2. 配置环境：
```

Comment:
[From AI Code Reviewer][bot] The removal of environment requirements and detailed installation steps could potentially hinder new users from setting up the project smoothly. Consider adding a simplified setup guide or a link to a detailed setup document.